tests/run_test.sh

```diff
@@ -130,11 +130,20 @@ atparse < ${PATHTR}/parm/${NEMS_CONFIGURE:-nems.configure} > nems.configure
 # Submit test job
 ################################################################################
 
-if [[ $ROCOTO = 'false' ]]; then
-  submit_and_wait job_card
+if [[ $SCHEDULER = 'none' ]]; then
```

Comment:
I believe you just facilitated my next job (run official regression tests in the cloud) by quite a bit!

Reply:
Cool. You will need lots of resources to run RT in the cloud!